apps/web/src/pages/EmployeeProfile/components/CareerDevelopmentSection/Display.tsx

```diff
@@ -0,0 +1,201 @@
+import { useIntl } from "react-intl";
+
+import {
+  commonMessages,
+  getExecCoachingInterest,
+  getMentorshipInterest,
+  getMoveInterest,
+  getOrganizationTypeInterest,
+} from "@gc-digital-talent/i18n";
+import {
+  EmployeeProfileCareerDevelopmentFragment,
+  EmployeeProfileCareerDevelopmentOptionsFragment,
+} from "@gc-digital-talent/graphql";
+import { Well } from "@gc-digital-talent/ui";
+import { empty, unpackMaybes } from "@gc-digital-talent/helpers";
+
+import ToggleForm from "~/components/ToggleForm/ToggleForm";
+import { hasAnyEmptyFields } from "~/validators/employeeProfile/careerDevelopment";
+import BoolCheckIcon from "~/components/BoolCheckIcon/BoolCheckIcon";
+
+import {
+  displayExecCoachingStatus,
+  displayMentorshipStatus,
+  getLabels,
+} from "./utils";
+
+interface DisplayProps {
+  employeeProfile: EmployeeProfileCareerDevelopmentFragment;
+  careerDevelopmentOptions: EmployeeProfileCareerDevelopmentOptionsFragment;
+}
```

Comment:
These should be something closer to `FragmentType<typeof EmployeeProfileCareerDevelopment_Fragment>`

Reply:
Refactored in [c3aaaf2](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12680/commits/c3aaaf22dc15805bb7efdf816ec2ec3e37dc76a1)